qualtran/bloqs/phase_estimation/lp_resource_state.py

```diff
@@ -94,9 +93,6 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
             }
         return ret
 
-    def _t_complexity_(self) -> 'TComplexity':
-        return TComplexity(rotations=self.bitsize + 1, clifford=2 + self.bitsize)
```

Comment:
@tanujkhattar isn't this TComplexity expression wrong? we use `self.bitsize` CRz gates but `TComplexity.rotations` is only for single qubit rotations, right?

Reply:
Yes, this is totally wrong.  Even if we revert the change to fix symbolics, this expression still needs to change.

Reply:
Reverted the _t_complexity_ removal, and added a comment.  Let me know if I misrepresented the factor in the comment.

Reply:
maybe we should make this change in a different PR? because it's a bug-fix for the tcomplexity expression, unlike the other changes that remove `_t_complexity_`

Reply:
I don't think that's necessary